sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java

```diff
@@ -800,6 +1262,49 @@ private Mono<PollResponse<AnalyzeActionsOperationDetail>> processAnalyzedModelRe
         Response<AnalyzeJobState> analyzeJobStateResponse,
         PollResponse<AnalyzeActionsOperationDetail> operationResultPollResponse) {
 
+        LongRunningOperationStatus status = LongRunningOperationStatus.SUCCESSFULLY_COMPLETED;
+        if (analyzeJobStateResponse.getValue() != null && analyzeJobStateResponse.getValue().getStatus() != null) {
+            switch (analyzeJobStateResponse.getValue().getStatus()) {
+                case NOT_STARTED:
```

Comment:
is this for legacy or language API? if the latter you need to account for partiallySucceeded